types/office-js-preview/index.d.ts

```diff
@@ -1659,8 +1659,11 @@ declare namespace Office {
          * - DialogMessageReceived. Triggered when the dialog box sends a message to its parent.
          *
          * - DialogEventReceived. Triggered when the dialog box has been closed or otherwise unloaded.
+         * 
+         * @param eventType Must be either DialogMessageReceived or DialogEventReceived.
+         * @param handler Takes an object with a `message` property if eventType is DialogMessageReceived. Takes an object with an `error` property if eventType is DialogEventReceived.
```

Comment:
This description reads as if the `addEventHandler` function could take either a message or error, rather than taking a function that accepts those arguments.

Reply:
good catch